Source/Tools/LinuxEmulation/LinuxSyscalls/GdbServer.cpp

```diff
@@ -107,6 +108,16 @@ GdbServer::GdbServer(FEXCore::Context::Context* ctx, FEX::HLE::SignalDelegator*
         return false;
       }
 
+      auto ThreadObject = FEX::HLE::ThreadManager::GetStateObjectFromFEXCoreThread(Thread);
+      ThreadObject->GdbInfo.Signal = Signal;
+
+      ThreadObject->GdbInfo.SignalPC = ArchHelpers::Context::GetPc(ucontext);
+      uint32_t IgnoreMask = Thread->CurrentFrame->InSyscallInfo & 0xFFFF;
```

Comment:
What does the magic 0xffff value do here?

Reply:
Nothing, I removed it.